Comment component

drillprop opened the pull request:
Fix #2452 

Designs: 
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2411%3A217092

The component is available in the storybook. Be aware that the reactions footer isn't in the scope of this PR.

drillprop committed:
create Comment component

rafalpawlow commented on packages/atlas/src/components/_comments/Comment/Comment.styles.ts:
```diff
@@ -0,0 +1,70 @@
+export const CommentHeaderDot = styled.div`
+  background-color: ${cVar('colorBackgroundAlpha')};
+  border-radius: 100%;
+  width: ${sizes(1)};
```
if I remember correctly we decided to not use `sizes` for width and height. Also, just a reminder that we have `square` helper 😛

rafalpawlow commented on packages/atlas/src/components/_comments/Comment/Comment.stories.tsx:
```diff
@@ -0,0 +1,33 @@
+      </BrowserRouter>
+    ),
+  ],
+  argTypes: {
```
Let's remove unused args like: `onEditLabelClick` `onEditClick` `onDeleteClick`

rafalpawlow commented on packages/atlas/src/components/_comments/Comment/Comment.tsx:
```diff
@@ -0,0 +1,142 @@
+  comment?: string
+  loading?: boolean
+  isEdited?: boolean
+  type: 'default' | 'deleted' | 'options'
```
I'm thinking, shouldn't we use some `isOwner` flag instead of `options` type? WDYT?

drillprop replied:
I'm not sure, I would rather stick to the types defined by designers in this case. Remember that also the creator of the video can delete a comment.

drillprop committed:
CR fixes add isAbleToEdit prop

toiletgranny commented:
It was a pleasure reviewing this! @drillprop thank you for taking the time to put together such a fantastic Storybook page for this component, it made the process so much easier!

I have a few thoughts/questions for you to consider:
1. I might be missing something, but for now it seems like we have an unnecessary 8px of a bottom margin underneath the comment's content. `.css-19idom { margin-bottom: 8px; }` Or is it because of the reactions footer that's not there yet?
2. I realize this might not be relevant at the moment, but just in case: let's keep in mind that the timestamp should be a hyperlink to that comment.
3. Ideally, there's a 1-second delay for displaying a tooltip with the exact timestamp. Currently, the tooltip is displayed instantly. Is this something we can add?
4. I might have missed that in the documentation, so apologies for that in advance, but ideally, both the avatar and the member handle/name should be hyperlinks to that member profile page. Is this something we can add?

drillprop committed:
add delay

drillprop committed:
remove unnecessary margin

drillprop committed:
added links, wrap story in MemoryRouter

drillprop commented:
@toiletgranny 
1. Removed
2. For now it's impossible to implement because we only have this component available in the storybook. But I added a "to-do" comment in the code, to not forget about this. 
3. Added
4. Added(but it might be hard to test this in the storybook)
